subprojects/docs/src/docs/userguide/authoring-builds/more_about_tasks.adoc

```diff
@@ -124,51 +134,50 @@ include::{snippetsPath}/tasks/accessUsingPath/tests/accessUsingPath.out[]
 ====
 include::sample[dir="snippets/tasks/configureUsingBlock/groovy",files="build.gradle[tags=declare-task]"]
 include::sample[dir="snippets/tasks/configureUsingBlock/kotlin",files="build.gradle.kts[tags=declare-task]"]
 ====
 
-This creates a copy task with no default behavior. The task can be configured using its API (see link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy]). The following examples show several different ways to achieve the same configuration.
+This registers a copy task with no default behavior.
+The task can be configured using its API (see link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy]).
+The following examples show several different ways to achieve the same configuration.
 
-Just to be clear, realize that the name of this task is “`myCopy`”, but it is of _type_ “`Copy`”. You can have multiple tasks of the same _type_, but with different names. You'll find this gives you a lot of power to implement cross-cutting concerns across all tasks of a particular type.
+Just to be clear, realize that the name of this task is `myCopy`, but it is of _type_ `Copy`.
+You can have multiple tasks of the same _type_, but with different names.
+You'll find this gives you a lot of power to implement cross-cutting concerns across all tasks of a particular type.
 
-.Configuring a task using the API
+.Configuring a task
 ====
 include::sample[dir="snippets/tasks/configureUsingVar/groovy",files="build.gradle[tags=configure]"]
 include::sample[dir="snippets/tasks/configureUsingVar/kotlin",files="build.gradle.kts[tags=configure]"]
 ====
 
-This is similar to the way we would configure objects in Java. You have to repeat the context (`myCopy`) in the configuration statement every time. This is a redundancy and not very nice to read.
-
-There is another way of configuring a task. It also preserves the context and it is arguably the most readable. It is usually our favorite.
+You can also store the task reference in a variable and use to configure the task further at a later point in the script.
 
-.Configuring a task using a DSL specific syntax
+.Retrieve a task reference and use it to configuring the task
 ====
 include::sample[dir="snippets/tasks/configureUsingBlock/groovy",files="build.gradle[tags=configure]"]
 include::sample[dir="snippets/tasks/configureUsingBlock/kotlin",files="build.gradle.kts[tags=configure]"]
 ====
 
-This works for _any_ task. Task access is just a shortcut for the `tasks.named()` (Kotlin) or `tasks.getByName()` (Groovy) method. It is important to note that blocks used here are for _configuring_ the task and are not evaluated when the task executes.
-
 Have a look at link:{javadocPath}/org/gradle/api/tasks/TaskContainer.html[TaskContainer] for more options for configuring tasks.
 
+[TIP]
+====
+If you use the _Kotlin DSL_ and the task you want to configure was added by a plugin, you can use a convenient accessor for the task.
+That is, instead of `tasks.named("test")` you can just write `tasks.test`.
```

Comment:
Doesn't this work with Groovy as well?

Reply:
In Groovy DSL you can _always_ use such a shortcut notation, because we dynamically resolve it to the task name. The generated accessors are not used in Groovy DSL.

However, for historical reasons, using this notation in Groovy is **not** lazy and will give you the Task instead of the TaskProvider.

Because we want to promote the lazy API, I don not mention it here. We should consider changing it in the future but we need to check how breaking such a change would be and when/if we could do it.